docs/root/configuration/http/http_conn_man/response_code_details.rst

```diff
@@ -0,0 +1,93 @@
+.. _config_http_conn_man_details:
+
+Response Code Details
+=====================
+
+If _%RESPONSE_CODE_DETAILS%_ is configured on in access logging, every log entry should include the detailed reason that stream ended.
+Below are the list of reasons the HttpConnectionManager or Router filter may send responses or reset streams.
```

Comment:
Should we mention that this list is not exhaustive because filters can respond with its own response codes? Optimally we would document these in the per-filter configs also, but that could be done later.